spk/syncthing/src/ca_reloader.sh

```diff
@@ -8,7 +8,7 @@ fi
 SYNCTHING=/var/packages/syncthing/target/bin/syncthing
 
 case $1 in
-    syncthing_webui)
+    syncthing-webui)
```

Comment:
This doesn't work because it does not match the definition in `syncthing.sc`.  There had been some discussion whether a hyphen or underscore is the correct separator, which ended in using the underscore consistently.  Please stick with that to avoid troubles during upgrades when DSM doesn't associate the same service again.

Reply:
This is only the paramerter of this script. It has no other meaning.
The usage tells that you have to call the script with the parameter "syncthing-webui" but the script worked only with the paramer "syncthing_webui".
Should I fix this the other way round? (keep the parameter syncthing_webui and fix the usage output)

Reply:
Oh now I see. Yes, definitely the script does work and it is called by DSM with the name containing an underscore.  Please fix the usage message instead.

Reply:
Oh no **I** see, didn't know that this script is called by DSM...